### DISCUSSION TITLE
Table alignment in markdown table

### DISCUSSION THREAD

**Discussion by maxjeblick** [h2oai/wave#2105] [Q&A]
HI,

I wonder if it is possible to top align text in a table (currently, it is center aligned). Rows are rendered in markdown.

![Screenshot 2023-07-27 at 09-51-00 H2O LLM Studio](https://github.com/h2oai/wave/assets/24281881/3937ad4b-3182-462d-9e6d-7f1def2e2de4)

**Comment by mturoci** [comment]
There is not besides custom CSS. Wondering why you want to use a table though. Wouldn't a dedicated detail page be better considering the amount of information you are trying to convey?

**Comment by maxjeblick** [comment]
> Wouldn't a dedicated detail page be better

Do you have any example code in mind? I'm happy for suggestions! The usecase is to improve the insights view in H2O LLM Studio, where we actually don't expect to see much code, on average.


![254880815-eb6f0c99-df06-42de-9d8f-143f2b0f83d1](https://github.com/h2oai/wave/assets/24281881/c046cdb1-aa93-44ca-b71a-94aa2ce06a3a)

**Reply by mturoci** [reply]
Your previous screenshot seems to have more cols (tokenized_text etc.). There are 2 options I would consider:

1. Render a selectable table on the LHS with a few short columns and a detail box on the RHS of the table. Row selection should update the detail box with the current data.
2. If there are multiple single-row columns (input_text, metrics and whatnot) and the need more space, render the table to the full width and create a master-detail view - once the row is clicked, show a completely new detail page with all the text.

This approach would not only make your table more compact and quicker to scan but also provide a better UX when reading the target/predicted texts since they will not need to fit into narrow columns, resulting in excessive linebreaks. 

Thoughts?

**Reply by maxjeblick** [reply]
Will check it out, thanks for the input!
